airbyte-cdk/python/airbyte_cdk/sources/declarative/declarative_component_schema.yaml

```diff
@@ -1001,19 +1692,31 @@ definitions:
         type: string
         enum: [RequestOption]
       field_name:
+        title: Configures which key should be used in the location that the descriptor is being injected into
```

Comment:
I guess that should be the `description`?

Reply:
yes, fixed